checker/tests/upperbound/IntroAdd.java

```diff
@@ -1,7 +1,7 @@
 import org.checkerframework.checker.upperbound.qual.*;
```

Comment:
These tests were, I think, meant to check that the checker behaves correctly when the user writes a non-sensical annotation. So the change to add "banana" as a parameter seems not to be consistent with their intent.